python/ct/client/log_client.py

```diff
@@ -196,8 +196,9 @@ def _parse_consistency_proof(response, servername):
 # A class that we can mock out to generate fake responses.
 class RequestHandler(object):
     """HTTPS requests."""
-    def __init__(self, connection_timeout=60):
+    def __init__(self, connection_timeout=60, ca_bundle=True):
```

Comment:
ca_bundle doesn't sound like something you'd pass as a boolean value.
is it really a flag?